hrcase/hrcase.php

```diff
@@ -447,10 +450,16 @@ function hrcase_getActionsSchedule($getNamesOnly = FALSE) {
 /**
  * function to check if the activity is created by task and assignments extension
  *
- * @param int $activity_type_id
+ * @param int $activity_id
  * @return boolean
  */
-function activityCreatedByTaskandAssignments($activity_type_id) {
+function activityCreatedByTaskandAssignments($activity_id) {
+  // PCHR-1177 - Status update from task lists caused a null activity type id to passed as parameter for this method.
+  // Passed activity_id instead to load data from db instead.
+  $params = array('id' => $activity_id);
+  $taskInDB = CRM_Activity_BAO_Activity::retrieve($params);
```

Comment:
The `$taskInDB` name sounds a bit odd to me. What if, for some reason, the task is not stored on DB anymore? This is just an infrastructure details, so I don't think it should be part of the name. Probably it would be better to just name it as $activity, after all, the method name is `activityCreatedByTaskandAssignments()`